google/cloud/security/common/gcp_type/resource.py

```diff
@@ -55,8 +55,8 @@ class ResourceType(object):
     # AppEngine
     APPENGINE = 'appengine'
 
-    # GKE
-    GKE = 'gke'
+    # KEngine
+    KENGINE = 'ke'
```

Comment:
This resource is really a cluster. It should be KE_CLUSTER

Reply:
Renamed to KE_CLUSTER